hack/make.sh

```diff
@@ -142,7 +142,7 @@ if [ -z "$DOCKER_DEBUG" ]; then
 fi
 
 LDFLAGS_STATIC=''
-EXTLDFLAGS_STATIC='-static'
+#EXTLDFLAGS_STATIC='-static'
 # ORIG_BUILDFLAGS is necessary for the cross target which cannot always build
```

Comment:
let's try if we can use `make dynamic-binary` ? and if it work fine, we can leave this line as-is.

Reply:
Done.